MIP101/MIP101.md

```diff
@@ -413,7 +413,7 @@ There must always be clear justification for all Alignment Artifact strengthenin
 
 ###### 2.5.1.2.2
 
-The Alignment Artifact strengthening work of AVCs must properly take into account and justify its impact on Future Alignment Risk of the AVC.
```

Comment:
Let's leave this as is. The Author intends to emphasize this phrase.

Reply:
Bumping this to prevent accidental merging.

Reply:
@blimpa please reverse this edit; it should not be part of the PR.